chat-commands.js

```diff
@@ -2265,6 +2265,25 @@ exports.commands = {
 	},
 	showblacklishelp: ["/showblacklist OR /showbl - show a list of blacklisted users in the room"],
 
+	sharedip: function (target, room, user) {
+		if (!this.can('ban')) return false;
+		if (!target) return this.errorReply("No IP entered.");
+		let parts = target.split(',');
```

Comment:
You can't split like this. :|

I guess maybe I should make a `splitOne` command or something...